packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -473,6 +472,41 @@ export class Users {
       }
     );
 
+    // Set unclaimedProceeds & unclaimedProfit
+    const marketsData = await db.findMarkets({
+      selector: { market: { $in: marketIds } },
+    });
+    for (const marketData of marketsData) {
+      marketTradingPositions[marketData.market].unclaimedProceeds = '0';
+      marketTradingPositions[marketData.market].unclaimedProfit = '0';
+      if (marketData.reportingState === MarketReportingState.Finalized || MarketReportingState.AwaitingFinalization) {
+        if (marketData.tentativeWinningPayoutNumerators) {
+          const outcome = marketData.tentativeWinningPayoutNumerators.findIndex((item: string) => Number(item) > 0);
+          const tokenBalanceChangedLogs = await db.findTokenBalanceChangedLogs(
```

Comment:
We should be doing a single query for all markets above are arranging it into a mapping so we can get share balances for each non zero winning outcome